__mocks__/aws-sdk.js

```diff
@@ -21,6 +24,57 @@ class SecretsManagerMock {
   }
 }
 
+class CloudFormationMock {
+  constructor() {
+    this.stacks = {};
+    this.timesCalled = 0;
+  }
+
+  describeStacks({ StackName }) {
+    this.timesCalled += 1;
```

Comment:
Is something like [this](https://jestjs.io/docs/en/expect#tohavebeencalledtimesnumber) useful?